wiki/osu!stream/en.md

```diff
@@ -364,8 +365,6 @@ Because of the difficulty in obtaining licensing for the said song.
 
 Head to the "Store" section in the game or pressing the "Download more songs..." at the "Play" section.
 
-The beatmaps are in bundle, and some are priced. Make sure to look out of the price tag before buying to prevent *accidental purchase*.
```

Comment:
Keep but change "are" to "were" and mention the blog post announcing it would be free?